blackberggroup/VA-iEX
# feat: new coming soon page; update iex menu/link

## What type of PR is this? (check all applicable)


## Description
Create new iEX 2024 "Coming Soon" page based upon home page that just includes the hero and CTA below the hero
Adjust iEX menu from "iEX 2024" to "iEX"

## Related Tickets & Documents
- Monday Task # 7206944561
- https://blackberg-group.monday.com/boards/1931719392/pulses/7206944561

## QA Instructions, Screenshots, Recordings
Confirm new "Coming Soon" page renders correctly and menu has been updated
dev-innovation.bgsandbox.com/hil/views/iex-2024/coming-soon.html
Confirm iEX menu is "iEX" and no longer "iEX 2024" and links to new iEX 2024 page /hil/views/iex-2024

### UI accessibility checklist
_If your PR includes UI changes, please utilize this checklist:_


## Are there any post deployment tasks we need to perform?
- Add GA code on production
- Proper meta tags utilized 

## Changed Files in this Pull Request ##
- _site/coming-soon.html
 - _site/coming-soon/index.html
 - coming-soon.html
 - .github/workflows/deploy-dev-sandbox.yml
 - .github/workflows/deploy-sandbox.yml
 - README.md
